src/sql/orderbook/prod_order_rewards.sql

```diff
@@ -110,12 +142,25 @@ order_protocol_fee_prices AS (
         opf.auction_id,
         opf.protocol_fee,
         opf.protocol_fee_token,
-        ap.price / pow(10, 18) as protocol_fee_native_price
+        CASE
```

Comment:
Are these used anywhere? Or did you add them for completeness purposes?

Reply:
They are not used anywhere. This was mostly to make it more or less a copy paste from the solver-rewards repo. Unfortunately there is some deviation anyways (e.g. the solver-rewards query does not keep `order_uid` as a column till the end.

Generally I would be in favor of making the queries as close as possible. So we should imho rather add used information to solver-rewards than remove unused info here.